test/plugins/features/movies/controller.test.js

```diff
@@ -69,4 +76,34 @@ describe('movie controller', () => {
 
   });
 
+  describe('add locations to movies', () => {
+
+    it('tries to add location to a non-existent movie', () => {
+      const payload = {
+        city: 'San Francisco',
+        state: 'California'
+      };
+
+      const id = 233444; // non-existent movie ID
```

Comment:
Perhaps you could use an id like 'non-existent id' or something like that to make it more clear that this id is invalid? Not sure if a data type error will get thrown as a result, but if not might makes sense to do so.

Reply:
The `id` field is an integer and hence, I had to pick an `id` that wasn't in the DB.